src/test/java/com/wcc/platform/service/CmsServiceIntegrationTest.java

```diff
@@ -14,7 +14,7 @@ class CmsServiceIntegrationTest {
   @Autowired private CmsService service;
 
   @Test
-  void getTeamPageTest() {
+  void testGetTeamPageTest() {
```

Comment:
Naming convention for the method can be improved, twice mentioning "test" and it seems like we testing the test class not the actual implementation class.